spec.html

```diff
@@ -21877,9 +21894,9 @@ <h1>Static Semantics: BoundNames</h1>
         AsyncFunctionDeclaration : `async` `function` `(` FormalParameters `)` `{` AsyncFunctionBody `}`
       </emu-grammar>
       <emu-alg>
-        1. Return &laquo; *"\*default\*"* &raquo;.
+        1. Return &laquo; ~default~ &raquo;.
```

Comment:
this seems like a good editorial PR to land separately and first :-)

Reply:
I'm only doing this here because it needs to be done for this PR, feel free to rip it out to a separate PR.

Reply:
Went ahead and filed #2155.